mobify/platform-scaffold
# Account Navigation Items

(brief description / PR title)

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1449
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/743 (**We need a SDK release**)

## Changes
- re-order menu items to have account links at the bottom

## Todos

## How to test-drive this PR
- checkout `WEB-1449__navigation` in the sdk
- run `npm link` (in SDK)
- run `npm install` (is SDK)
- checkout this branch 
- run `npm link progressive-web-sdk`
- run `npm run dev`
- [Preview](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Open menu and check the links
